lib/msal-core/package.json

```diff
@@ -28,50 +28,60 @@
     "clean": "shx rm -rf dist docs lib-commonjs lib-es6",
     "doc": "typedoc --out ./docs ./src/ --gitRevision dev",
     "build:modules": "tsc && tsc -m es6 --outDir lib-es6",
-    "build": "npm run clean && npm run doc && npm run build:modules && npm run lint && npm run build:webpack && npm test",
-    "test": "karma start && jasmine tests/MSALSpec.js",
+    "build": "npm run clean && npm run doc && npm run build:modules && npm run lint && npm run build:webpack && npm run test",
+    "jtest": "typings install && karma start && jasmine-node --nohelpers tests/MSALSpec.js",
```

Comment:
what is jtest?

Reply:
We did not take out jasmine yet and hence this is just fallback. Will go away once all UT is ready.

Reply:
This typings is unneeded in this call anyway, it doesn't do anything.

Reply:
I have taken out 'run' in my latest commit.